compressor/tests/test_utils.py

```diff
@@ -44,3 +46,15 @@ def test_finders_is_none_if_staticfiles_is_not_installed(self):
                 self.assertTrue(compressor.utils.staticfiles.finders is None)
         finally:
             reload(compressor.utils.staticfiles)
+
+
+class TestGetClass(TestCase):
+
+    def test_get_class(self):
+        try:
+            get_class('common.uglify.JsUglifySourcemapCompressor')
+        except FilterError as e:
+            self.assertTrue(('Failed to import common.uglify.JsUglifySourcemapCompressor. '
+                             'ImportError is: No module named' in str(e)))
+        else:
+            self.fail('FilterError not raised')
```

Comment:
check out this [stackoverflow answer](http://stackoverflow.com/a/3166985/4312944) which should make this prettier.